static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+          example: "{API_KEY}"
+    validateTagsResponse:
+      type: object
+      properties:
+        invalidTags:
+          type: array
+          description: "An array listing the invalid tags."
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                example: "invalid-tag"
+        validTags:
+          type: array
+          items: 
+            type: object
+            description: "An array listing the valid tags."
+            properties:
+              id:
+                type: string
+                example: "d113f40c-0097-4626-8d5f-6d5017694453"
+          example:
+            - id: d113f40c-0097-4626-8d5f-6d5017694453
+            - id: 2bd5ddd9-7284-4767-81d9-c75b122f2a6a
+  parameters: 
+    folder-type:
+      name: FOLDER_TYPE
+      in: path
+      description: The type of folder you're retrieving. Supported values include `segments` and `datasets`.
+      required: true
+      schema:
+        type: string
+    folder-id:
+      name: FOLDER_ID
+      in: path
+      description: The ID of the folder you're retrieving.
+      required: true
+      schema:
+        type: string
+    authorization:
+      name: Authorization
+      in: header
+      description: The access token which can be copied from your Experience Platform integration, prefixed with "Bearer ". For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+      required: true
+      schema:
+        type: string
+    x-gw-ims-org-id:
+      name: x-gw-ims-org-id
+      in: header
+      description: The Organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+      required: true
+      schema:
+        type: string
+    x-api-key:
+      name: x-api-key
+      in: header
+      description: The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+      required: true
+      schema:
+        type: string
+    x-sandbox-name:
+      name: x-sandbox-name
+      in: header
+      description: All resources in Experience Platform are isolated to specific virtual sandboxes. Requests to Platform APIs require a header that specifies the name of the sandbox in which the operation will take place. See the [sandboxes overview](https://adobe.com/go/sandbox-overview-en) for more information.
+      required: true
+      schema:
+        type: string
+    content-type:
+      name: Content-Type
+      in: header
+      description: Type of content being sent in the body of the request. Should be `application/json`.
```

Comment:
Really minor. Please can you make the second sentence complete:
"This value should be `application/json`."